.travis.yml

```diff
@@ -3,16 +3,12 @@ dist: trusty
 sudo: required
 
 php:
-  - 5.5
-  - 5.6
-  - 7.0
-  - hhvm
-  - hhvm-nightly
+  - 7.2
+  - nightly
 
 matrix:
   allow_failures:
-    - php: hhvm
-    - php: hhvm-nightly
+    - nightly
 
 before_script:
   - sudo add-apt-repository -y ppa:crate/stable
```

Comment:
Unrelated to this PR, but should also be fixed (probably before this PR is merged): The PPA is deprecated, CrateDB for Ubuntu is hosted on our own repo since 2.1.0
see: https://crate.io/docs/crate/guide/en/latest/deployment/linux/ubuntu.html

Reply:
As @Autophagy suggested, you can use Docker for CrateDB https://github.com/crate/crate-pdo/commit/20bc842ce9ce9b11bc344d8d0b495c465762e9c8#diff-354f30a63fb0907d4ad57269548329e3R12

Reply:
Just make a separate commit for that, then it can be within this PR actually.